tests/integration/framework/process/sentry/sentry.go

```diff
@@ -196,8 +200,14 @@ func (s *Sentry) HealthzPort() int {
 	return s.healthzPort
 }
 
+func (s *Sentry) Namespace() string {
+	return s.namespace
+}
+
 func (s *Sentry) TrustDomain(t *testing.T) string {
-	require.NotNil(t, s.trustDomain)
```

Comment:
The nil check here was deliberate so a trust domain was always set when a consumer using the sentry trust domain always sets one on the process. Can the consumer set the trust domain instead?

Reply:
This is done with .WithTrustDomain on the sentry process in the failing test.

Reply:
I think this was a change from a prior PR that was accidentally reverted bc I didn't remember it going thru and saw the diff and wanted to rm it. [Looks like you introduced it here in this PR](https://github.com/cicoyle/dapr/commit/67f84465d2672bca86c7aa74543ea177db579039#diff-e930ca5083fe83eb4c5e7f8cea733d7b943980e828a5511aae722bb6a25fa464R200)

Reply:
Thanks- not sure why I did that 🙂 
Ok, not a big deal

Reply:
makes sense to me to have dev experience be `m.sentry = sentry.New(t)` and auto set the trust domain, which this essentially does otherwise it errors on the trust domain being nil. I totally see what you are saying with 
```
	m.sentry = sentry.New(t, sentry.WithTrustDomain("localhost"))
```
Just one more step :)